### DISCUSSION TITLE
Googling public repository fails

### DISCUSSION THREAD

**Discussion by JMlisp** [community/community#121666] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Two days ago I created my first public repository named “og2vlm” in GitHub, which however, even if it is accessible via its URL https://github.com/jmlisp/og2vlm, it is not also found when googling in a browser for “og2vlm”. Does somebody know how this problem can be solved? Thanks a lot.

**Comment by gurjeetsinghvirdee** [comment, marked as answer]
hey @JMlisp 

Searching for a repository by its name directly in a search engine like Google might not yield immediate results unless the repository gains significant popularity. Generally, repositories only appear at the top of search results when they've earned a considerable number of stars and forks from contributors. To improve the visibility of your repository, you could promote it through various channels, such as sharing it on social media, participating in relevant communities, and encouraging others to star and contribute to it. Over time, as the repository gains traction and activity, it's more likely to become more visible in search results.

**Comment by JMlisp** [comment]
Thanks very much for the detailed and immediate response. The only question I would have anyway is: how can a repository gain popularity if it is not visible on the Internet at all?

**Reply by gurjeetsinghvirdee** [reply]
Creating projects that address real-world problems or fulfill specific needs within the developer community is crucial for gaining visibility and popularity, even if the repository isn't initially visible on the Internet. Originality and usefulness are key factors in attracting attention to your repository.

Additionally, you can keep an eye on trending projects by visiting the homepage of GitHub and checking the bottom right side, where trending repositories are showcased. Engaging with these projects, understanding what makes them popular, and incorporating similar elements into your own projects can also help increase visibility over time., when your repo

**Comment by JMlisp** [comment]
Thank you very much again, I will try to follow your precious suggestions, hoping to make the repository visible as soon as possible.

**Comment by JMlisp** [comment]
Thanks a lot for your precious suggestions.

**Reply by gurjeetsinghvirdee** [reply]
Most welcome 🤗
